Acacia keyboard (#19532)
qmk/qmk_firmware: keyboards/handwired/acacia/readme.md

# Acacia

A pseudo-split keyboard with 2+4x3+2 vertically staggered keys and 4+1 thumb keys for each hand.

* Keyboard Maintainer: [jpsecher](https://github.com/jpsecher)
* Hardware Supported: https://github.com/jpsecher/keyboards/acacia

> Make example for this keyboard (after setting up your build environment):
> 
>     make handwired/acacia
> 
> or
> 
>     qmk compile -kb handwired/acacia -km default
> 
> To clean up:
> 
>     qmk clean

Reviewer: ```suggestion
Make example for this keyboard (after setting up your build environment):

    make handwired/acacia:default

Flashing example for this keyboard:

    make handwired/acacia:default:flash
```
